components/schema-migrator/migrations/202001210900_system_auth_tenant_nullable.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE system_auths
```

Comment:
I think there should also be an update that updates all entries with value '00000000-0000-0000-0000-000000000000' to null value.